src/journey-maps/angular/journey-maps.ts

```diff
@@ -576,20 +576,17 @@ export class SbbJourneyMaps implements OnInit, AfterViewInit, OnDestroy, OnChang
 
           // remove previous data from map
           this._mapJourneyService.updateJourney(this._map, mapSelectionEventService, undefined);
-          this._mapTransferService.updateTransfer(this._map, undefined);
+          this._mapTransferService.updateTransfer(this._map, undefined); // TODO: asi, still needed for v1?
```

Comment:
Is this needed for v1/ api calls?

Reply:
I'm confused by the combination of the terms "still" and "v1". Did you perhaps mean "still needed for v2" ?